drivers/hmis/app/models/hmis/hud/custom_data_element.rb

```diff
@@ -25,12 +25,15 @@ class Hmis::Hud::CustomDataElement < Hmis::Hud::Base
     :value_json,
     :value_string,
     :value_text,
+    :value_file,
   ].freeze
 
   belongs_to :owner, polymorphic: true, optional: false
   belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
   belongs_to :user, **hmis_relation(:UserID, 'User'), optional: true, inverse_of: :custom_data_elements
   belongs_to :data_element_definition, class_name: 'Hmis::Hud::CustomDataElementDefinition', optional: false
+  belongs_to :value_file, class_name: 'Hmis::File', optional: true, autosave: true, foreign_key: 'value_file_id'
```

Comment:
This adds a relationship going from the CDE to the file, but I haven't configured anything in the opposite direction, so right now you can't easily get from the file back to the CDE. Would that be desirable?